src/SendGrid.Extensions.DependencyInjection/InjectableSendGridClient.cs

```diff
@@ -0,0 +1,20 @@
+﻿// <copyright file="InjectableSendGridClient.cs" company="Twilio SendGrid">
+// Copyright (c) Twilio SendGrid. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// </copyright>
+
+using System.Net.Http;
+using Microsoft.Extensions.Options;
+
+namespace SendGrid.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// A wrapped SendGridClient with single constructor to inject managed <see cref="HttpClient"/> by the <see cref="IHttpClientFactory"/>
```

Comment:
I personally think the wording here could be a little more verbose. Something along the lines of `[...] to inject an <see cref="HttpClient"/> whose lifetime is managed externally, e.g. by an IOC container.`

Reply:
Thanks, I personally prefer DI container.